### ISSUE TITLE
Update HARK dependency to 0.12.0

### ISSUE THREAD

**Issue by sbenthall** [econ-ark/DistributionOfWealthMPC#6]
Update HARK dependency to latest stable release. Currently, 0.12.0

**Comment by alanlujan91** [comment]
It is now working with master branch which will be released soon, so presumably it's working with 0.13 already in PR #4

**Comment by sbenthall** [comment]
Ok. Do the results on the master branch meet the substantive result requirement as described in #7 ?

What I was going to do is:
 1. Make the automated test for the substantive results #7 
 2. Use it to test the results using HARK 0.12.0. If passing, upgrade.
 3. Use the test to check 0.13.0 with your PR. If passing, upgrade by merging the PR.

But if the substantive results are all still good with master, then this may be more procedural than necessary.

**Comment by sbenthall** [comment]
I tested the results with HARK 0.12.0 with the test script manually.
Then I pushed the update to the requirements file.
I've cut a release, 1.0, for this version of the REMARK.

https://github.com/econ-ark/DistributionOfWealthMPC/releases/tag/1.0

I will now change the `master` branch so that it uses an unpinned version of `HARK`. This makes `master` into a 'development branch', which is not guaranteed to work.

External deployments of the REMARK, such as to hosted mybinder instances, should use the tagged version.
